AbdoulNuru/Teamwork
# #168803369 Add delete article endpoint

### What does this PR do?

Have the delete article route setup

### Description of task to be completed?

Add delete endpoint to facilitate users to delete articles they have created, if they are logged in.

### How should this be manually tested?

- Clone this repo to your local repository using `git clone <link>`
- Install all project dependencies using `npm install <package name>`
- In your command line use `npm start` to run the project
- Now using postman u can send a `delete` request to this url `/api/v1/articles/:id`
- Make sure in your request header you have *key: Content-Type value: application/json and 
  Authorization: Bearer token*

### What are the relevant pivotal tracker stories?

#168803369 https://www.pivotaltracker.com/story/show/168803369
